specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -564,621 +715,1772 @@
-      "description": "Numeric value representing an image that displays the `iconPhrase`. Please refer to [Weather Service Concepts](https://aka.ms/AzureMapsWeatherConcepts) for details.",
-      "enum": [
-        1,
-        2,
-        3,
-        4,
-        5,
-        6,
-        7,
-        8,
-        9,
-        10,
-        11,
-        12,
-        13,
-        14,
-        15,
-        16,
-        17,
-        18,
-        19,
-        20,
-        21,
-        22,
-        23,
-        24,
-        25,
-        26,
-        27,
-        28,
-        29,
-        30,
-        31,
-        32,
-        33,
-        34,
-        35,
-        36,
-        37,
-        38,
-        39,
-        40,
-        41,
-        42,
-        43,
-        44
-      ],
-      "x-ms-enum": {
-        "name": "IconCode",
-        "modelAsString": true,
-        "values": [
+    },
+    "/weather/tropical/storms/active/{format}": {
+      "get": {
+        "description": "**Get Tropical Active Storms**\n\n**Applies to**: Gen 1 (S0 and S1) and Gen 2 pricing tiers.\n\nGet all government-issued active tropical storms. Information about the tropical storms includes, government ID, basin ID, year of origin, name and if it is subtropical.",
+        "operationId": "Weather_GetTropicalActiveStorms",
+        "x-ms-examples": {
+          "Get Tropical Active Storms": {
+            "$ref": "./examples/GetActiveStorms.json"
+          }
+        },
+        "parameters": [
           {
-            "value": 1,
-            "name": "Sunny",
-            "description": "Sunny"
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
           },
           {
-            "value": 2,
-            "name": "MostlySunny",
-            "description": "Mostly Sunny"
+            "$ref": "#/parameters/ApiVersion"
           },
           {
-            "value": 3,
-            "name": "PartlySunny",
-            "description": "Partly Sunny"
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/JsonFormat"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ActiveStormResponse"
+            }
           },
-          {
-            "value": 4,
-            "name": "IntermittentClouds",
-            "description": "Intermittent Clouds"
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/weather/tropical/storms/{format}": {
+      "get": {
+        "description": "**Get Tropical Storm Search**\n\n**Applies to**: Gen 1 (S0 and S1) and Gen 2 pricing tiers.\n\nSearch government-issued tropical storms by year, basin ID, and government ID. Information about the tropical storms includes, government ID, basin ID, status, year, name and if it is subtropical.",
+        "operationId": "Weather_GetTropicalStormSearch",
```

Comment:
Probably needs an `x-ms-client-name`. Should be something like `SearchTropicalStorm` or `SearchTropicalStormData`

Reply:
Added